kratos/includes/master_slave_constraint.h

```diff
@@ -355,6 +355,24 @@ class MasterSlaveConstraint
         KRATOS_ERROR << "GetMasterDofsVector not implemented in MasterSlaveConstraintBaseClass" << std::endl;
     }
 
+    /**
+     * @brief This method resets the values of the slave dofs
+     * @param rCurrentProcessInfo the current process info instance
+     */
+    virtual void ResetSlaveDofs(const ProcessInfo& rCurrentProcessInfo)
```

Comment:
just to tell that these functions are actually candidate to be virtual pure ... (not now...)